test/image/mocks/gl3d_snowden_altered.json

```diff
@@ -24,8 +24,8 @@
                 "z": 2
             },
             "up": {
-                "x": 1,
-                "y": 0,
+                "x": 0,
+                "y": 1,
```

Comment:
So... `gl3d_ibm_plot` has the old (verrrry old) camera format, and converts to the new format with a tilted `up`, and you allowed it to shift into `orbit` mode; that's great. And `gl3d_surface_lighting` you converted to the new camera format and turntable-compatible `up`, so the image changed because it's somewhat different from the output of the previous old->new conversion; also great.

But what's going on with `gl3d_snowden_altered`? The image didn't change, but the change you made to `up` is *not* to `{x: 0, y: 0, z: 1}` so I'd have expected the image to change, since now it will render in orbit mode.